08LessonEight/homework/exercise-06.html

```diff
@@ -3,8 +3,42 @@
 
 <head>
     <title>Exercise 06</title>
+    <link type="text/css" rel="stylesheet" href="css/exercise-06-style.css" />
 </head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<body>
+    <main>
+        <section class="mainframe clearfix">
+            <div class="infobox clearfix">

```

Comment:
Regarding my Lesson 8 exercise-06-style.css -
The reason my infoboxes have clearfixes on them is so that my buttons will float left of the little red bar to the right of them. 

   

 On Saturday, August 27, 2016 9:52 PM, Erik Zambrano <notifications@github.com> wrote:
 

 In 08LessonEight/homework/exercise-06.html:> \ No newline at end of file
> +<body>
> +    <main>
> +        <section class="mainframe clearfix">
> +            <div class="infobox clearfix">
Again, you are using clearfix to the elements that are floating (same with elements on line 21 and 31)
clear-fix it's applied to the parent when all child elements are floating - not to childs—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.